src/database.ts

```diff
@@ -0,0 +1,15 @@
+import * as mongoose from "mongoose";
+
+class Database {
+  public db_connect(mongoUrl: string | undefined, mongoDb: string | undefined) {
+    mongoose.connect(`'${mongoUrl}/${mongoDb}'`,  { useNewUrlParser: true })
```

Comment:
Do we not need to return this?

Reply:
I didn't initially include { useNewUrlParser: true } but the following error is given when we try to connect to the MongoDB 

**(node:13420) DeprecationWarning: current URL string parser is deprecated, and will be removed in a future version. To use the new parser, pass option { useNewUrlParser: true } to MongoClient.connect.**

Reply:
Oh no sorry, I meant do we need to return the mongoose.connect statement so we can use the connection in another file?

Reply:
No I don't think so.